apps/04TowersOfHanoi.js

```diff
@@ -32,6 +30,8 @@ function movePiece(startStack, endStack) {
       //add that last element
         //to end stack
 
+        //pieceMoves++
+
 
 }
 
```

Comment:
Very good! In some programming languages, this feature would be called "peek"
```javascript
var lastElStartStack = startStackArray[startStackArray.length - 1];
```
and it would work the same wayy .push and .pop do